VideoLocker/src/main/java/org/edx/mobile/base/BaseFragmentActivity.java

```diff
@@ -403,7 +403,7 @@ public void setTitle(CharSequence title) {
                 if(titleTextView!=null){
                     titleTextView.setTextColor(getResources().getColor(R.color.edx_white));
                     titleTextView.setTypeface(type);
-                    bar.setTitle(title);
+                    bar.setTitle("  " + title);
```

Comment:
Hrrrm. I tried a similar thing in TextEdit, and it seemed to work, so probably it'll be fine, though it'd be great to check if you haven't already.